### PR TITLE
DM-40388-batchfix: Fix bps clustering configs to match calibrateImage adoption

### PR DESCRIPTION
**Pull request by BrunoSanchez**
Modified bps clustering yamls to group the right tasks.

### CODE REVIEW

File: bps/clustering/clustering_ApPipeWithFakes.yaml [hunk 1]
```diff
@@ -19,8 +19,7 @@
 clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.dimension_clustering
 cluster:
   singleFrame:
-    # TODO DM-40388: replace this with isr,calibrateImage
-    pipetasks: isr,characterizeImage,calibrate
+    pipetasks: isr,calibrateImage
```
Comment (parejkoj) [inline bps/clustering/clustering_ApPipeWithFakes.yaml:22]: These have to stay as-is: the Fakes pipeline isn't ready to use CalibrateImage. I just filed [DM-43351](https://rubinobs.atlassian.net/browse/DM-43351) for this. Please just change the TODO to reference that ticket.

File: bps/clustering/clustering_ApTemplate.yaml [hunk 2]
```diff
@@ -12,8 +12,7 @@
 clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.dimension_clustering
 cluster:
   singleFrame:
-    # TODO DM-40389: replace this with isr,calibrateImage
-    pipetasks: isr,characterizeImage,calibrate
+    pipetasks: isr,calibrateImage
```
Comment (parejkoj) [inline bps/clustering/clustering_ApTemplate.yaml:15]: Leave this all exactly as is.

File: bps/clustering/clustering_HSC_ApTemplate.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@
 clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.dimension_clustering
 cluster:
   singleFrame:
-    pipetasks: isr,characterizeImage,calibrate
+    pipetasks: isr,calibrateImage
```
Comment (parejkoj) [inline bps/clustering/clustering_HSC_ApTemplate.yaml:16]: Leave this all as is.

**Comment by BrunoSanchez** [comment]
Done with changes. Please check once more.

File: bps/clustering/clustering_HSC_ApTemplate.yaml (hunk 3 above)
Comment (kfindeisen) [reply bps/clustering/clustering_HSC_ApTemplate.yaml]: `ApTemplate` [does, in fact, use `calibrateImage`](https://github.com/lsst/ap_pipe/blob/main/pipelines/_ingredients/ApTemplate.yaml#L12).

File: bps/clustering/clustering_ApTemplate.yaml (hunk 2 above)
Comment (kfindeisen) [reply bps/clustering/clustering_ApTemplate.yaml]: `ApTemplate` [does, in fact, use `calibrateImage`](https://github.com/lsst/ap_pipe/blob/main/pipelines/_ingredients/ApTemplate.yaml#L12).

Comment (parejkoj) [reply bps/clustering/clustering_ApTemplate.yaml]: Oh, hmm, I was not intending on changing that, but now I see that the import there made it change.

Hopefully that's all moot, as we should be moving away from ApTemplate to just using the DRP pipeline.

**Review by parejkoj — APPROVED** [review]

**Pull request merged**